app/src/main/java/com/example/triptracker/view/home/HomeScreen.kt

```diff
@@ -58,9 +58,14 @@ import com.example.triptracker.viewmodel.HomeViewModel
  *
  * @param navigation: Navigation object to use for navigation
  * @param homeViewModel: HomeViewModel to use for fetching itineraries
+ * @param test: Boolean to test the function
  */
 @Composable
-fun HomeScreen(navigation: Navigation, homeViewModel: HomeViewModel = viewModel()) {
+fun HomeScreen(
+    navigation: Navigation,
+    homeViewModel: HomeViewModel = viewModel(),
+    test: Boolean = false
```

Comment:
This is a very ugly way to fix the problem but I mean if it works all good :)

Reply:
I knowwwwwww. Tried to put the  `readyToDisplay`and `profile` in the UserProfileViewModel but it's harder than I thought and it didn't work when I tried... will still try to fix this later